bin/sozo/src/commands/migrate.rs

```diff
@@ -24,13 +24,6 @@ pub struct MigrateArgs {
     #[command(subcommand)]
     pub command: MigrateCommand,
 
-    #[arg(long, global = true)]
-    #[arg(help = "Name of the World.")]
-    #[arg(long_help = "Name of the World. It's hash will be used as a salt when deploying the \
-                       contract to avoid address conflicts. If not provided root package's name \
-                       will be used.")]
-    name: Option<String>,
```

Comment:
I was wondering if it makes sense to allow a seed to be passed as argument. But I guess it's better this way, defined one in the profile, and then the user just rely on profile. 👍

Reply:
yeah allowing it to be passed from cli would overwrite the manifests of the default profile, it can be easy to forget passing `-P` flag so i think its better to not allow it